geolocation/enabled.https.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset="utf-8"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js?feature=bidi"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+    promise_setup(async () => {
+        // Ensure permission is granted before proceeding.
+        await test_driver.bidi.permissions.set_permission({
+            descriptor: {name: "geolocation"},
+            state: "granted",
+        });
+
+        // Ensure any previously set geolocation emulations are cleared.
+        await test_driver.bidi.emulation.set_geolocation_override(
```

Comment:
I don't think we have agreement on the shape of the BiDi API? @reillyeon?

Reply:
The API is specified here:
* https://www.w3.org/TR/geolocation/#emulation
* https://www.w3.org/TR/webdriver-bidi/#command-emulation-setGeolocationOverride

Reply:
https://github.com/w3c/geolocation/pull/183 was merged based on consensus between Chromium and Firefox.